### PR TITLE
Minor change: changed var to const (docs/Web/JavaScript/Reference/Global_Objects/bigint64array/bigint64array)

### PR DESCRIPTION
**Pull request by tech-writer-jill**
docs/Web/JavaScript/Reference/Global_Objects/bigint64array/bigint64array

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/bigint64array/bigint64array/index.md [hunk 1]
```diff
@@ -64,28 +64,28 @@ new BigInt64Array(buffer, byteOffset, length);
...
-var buffer = new ArrayBuffer(32);
-var z = new BigInt64Array(buffer, 0, 4);
+const buffer = new ArrayBuffer(32);
+const z = new BigInt64Array(buffer, 0, 4);
 
 // From an iterable
-var iterable = function*(){ yield* [1n, 2n, 3n]; }();
-var bigint64 = new BigInt64Array(iterable);
+const iterable = function*(){ yield* [1n, 2n, 3n]; }();
+const bigint64 = new BigInt64Array(iterable);
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/bigint64array/bigint64array/index.md:88]: This is an interesting case where the example is redeclaring the same variable, which is allowed with `var` but not with `let` or `const`. So we would need to give the second one a different name, like:

```suggestion
const bigint64_2 = new BigInt64Array(iterable);
```

(I'm not terribly happy with the `_2` suffix here but I think it is OK in this example.)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for you PR, @tech-writer-jill ! In this case the example is redeclaring the variable, which with `const` or `let` is not allowed, so we would need to use a different name for the second declaration.

**Review by tech-writer-jill — COMMENTED** [review]
Thanks for clarification on needed correction.

**Review by wbamberg — APPROVED** [review]
👍 thank you for the update, @tech-writer-jill !

**Pull request merged**